src/services/jwt.service.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const { id, email } = user;
+  const token = jwt.sign({ id, email }, process.env.JWT_KEY);
```

Comment:
Consider adding an expiration time to the JWT token using the `expiresIn` option in `jwt.sign`. This will enhance security by ensuring tokens are valid only for a limited time.